### PR TITLE
[#2120] Update RepoSense contributors in documentation

### LINKED ISSUE #2120: Update RepoSense contributors in documentation
Update the [About](https://reposense.org/about.html) page in the RepoSense documentation site to reflect the new current team and past members list.

### PR DESCRIPTION
**Pull request by asdfghjkxd**
Fixes #2120

### Proposed commit message

```
The current About page on the RepoSense docs does not reflect the
updated list of developers working on RepoSense.

Let's move to update the list to more accurately reference the current
developers of RepoSense.
```

### Other information

I'm not too sure what the start dates for everyone's contributions, so please do update me if I got it wrong!

### CODE REVIEW

File: docs/about.md [hunk 1]
```diff
@@ -55,16 +55,51 @@ Contributor [2021 January - ]<br/>
...
 
-### [Gokul Rajiv](https://github.com/gok99)
+### [Zhou Jiahao](https://github.com/zhoukerrr)
 **Role**: Contributor [2022 January - ]<br/>
 
 <!-- ------------------------------------------------------------------------------------------------------ -->
 
-### [Zhou Jiahao](https://github.com/zhoukerrr)
+### [Charisma Kausar](https://www.github.com/ckcherry23)
+**Role**: Contributor [2024 January - ]<br/>
```
Comment (ckcherry23) [inline docs/about.md:64]:
```suggestion
**Role**: Committer [2024 January - ]<br/>
Contributor [2022 December - ]<br/>
```

File: docs/about.md [hunk 2]
```diff
@@ -55,16 +55,51 @@ Contributor [2021 January - ]<br/>
...
 
 <!-- ------------------------------------------------------------------------------------------------------ -->
 
+### [Marcus Tang Xin Kye](https://www.github.com/MarcusTXK)
+**Role**: Contributor [2024 January - ]<br/>
+
+<!-- ------------------------------------------------------------------------------------------------------ -->
+
+### [Alvis Ng](https://www.github.com/supermii2)
+**Role**: Developer [2024 January - ]<br/>
```
Comment (ckcherry23) [inline docs/about.md:84]: Use 'Contributor' instead of 'Developer'
```suggestion
**Role**: Contributor [2024 January - ]<br/>
```

Comment (asdfghjkxd) [reply docs/about.md:84]: Got it! Will update and commit!

File: docs/about.md (hunk 1 above)
Comment (asdfghjkxd) [reply docs/about.md:64]: Would it be safe to assume that all current mentors have the same title? (except for maybe @gok99 who was already on the list?)

Comment (ckcherry23) [reply docs/about.md:64]: Yes, we should have the same title. Even @gok99 would be a Committer now. But his contributor start date should be a year earlier.

Comment (asdfghjkxd) [reply docs/about.md]: I see, I have updated the commit now, it should reflect the correct roles and start dates now!

File: docs/about.md [hunk 3]
```diff
@@ -55,13 +55,50 @@ Contributor [2021 January - ]<br/>
...
+<!-- ------------------------------------------------------------------------------------------------------ -->
+
+### [Gokul Rajiv](https://github.com/gok99)
+**Role**: Committer [2024 January - ]<br/> 
+Contributor [2021 January - ]<br/>
+
+<!-- ------------------------------------------------------------------------------------------------------ -->
+
+### [Marcus Tang Xin Kye](https://www.github.com/MarcusTXK)
+**Role**: Comitter [2024 January - ]<br/>
```
Comment (jonasongg) [inline docs/about.md:81]: seems to be a typo here!

**Review by jonasongg — APPROVED** [review]

File: docs/about.md [hunk 4]
```diff
@@ -55,13 +55,50 @@ Contributor [2021 January - ]<br/>
...
-### [Zhou Jiahao](https://github.com/zhoukerrr)
-**Role**: Contributor [2022 January - ]<br/>
+### [Charisma Kausar](https://www.github.com/ckcherry23)
+**Role**: Committer [2024 January - ]<br/>
+Contributor [2022 December - ]<br/>
+
+<!-- ------------------------------------------------------------------------------------------------------ -->
+
+### [David Gareth Ong](https://www.github.com/vvidday)
+**Role**: Committer [2024 January - ]<br/>
```
Comment (ckcherry23) [inline docs/about.md:70]: All mentors were contributors too! (So update for David and Marcus)
```suggestion
**Role**: Committer [2024 January - ]<br/>
Contributor [2023 January - ]<br/>
```

File: docs/about.md [hunk 5]
```diff
@@ -55,13 +55,50 @@ Contributor [2021 January - ]<br/>
 
 <!-- ------------------------------------------------------------------------------------------------------ -->
 
-### [Gokul Rajiv](https://github.com/gok99)
+### [Zhou Jiahao](https://github.com/zhoukerrr)
 **Role**: Contributor [2022 January - ]<br/>
```
Comment (ckcherry23) [inline docs/about.md:59]: Add "Committer [2023 January - ]" for Zhou Jiahao, Chan Jun Da, Tay Yi Hsuen
```suggestion
**Role**: Committer [2023 January - ]<br/>
Contributor [2022 January - ]<br/>
```

**Review by ckcherry23 — COMMENTED** [review]
Hi @asdfghjkxd, based on the data we have for now, I have left 2 comments to look into.

File: docs/about.md (hunk 3 above)
Comment (asdfghjkxd) [reply docs/about.md:81]: Oh oops missed out a `m`, will fix!

File: docs/about.md (hunk 4 above)
Comment (asdfghjkxd) [reply docs/about.md:70]: I see, will do!

File: docs/about.md (hunk 5 above)
Comment (asdfghjkxd) [reply docs/about.md:59]: Sure thing!

File: docs/about.md [hunk 6]
```diff
@@ -46,22 +46,64 @@ Contributor [2021 January - ]<br/>
...
+### [David Gareth Ong](https://www.github.com/vvidday)
+**Role**: Committer [2024 January - ]<br/>
+Contributor [2023 January - ]<br/>
 
 <!-- ------------------------------------------------------------------------------------------------------ -->
 
 ### [Gokul Rajiv](https://github.com/gok99)
-**Role**: Contributor [2022 January - ]<br/>
+**Role**: Committer [2024 January - ]<br/> 
+Contributor [2021 January - ]<br/>
```
Comment (gok99) [inline docs/about.md:80]: This should be 2022 January (same as Jiahao)

**Review by ckcherry23 — CHANGES_REQUESTED** [review]
Hi @asdfghjkxd, Can I trouble you to make some more updates to the contributors list? 

1. We got information from prof about some members and their roles:

```
Wang Chao
Role: Project Lead [2021 Jan - 2021 Dec]
Maintainer [2020 May - 2021 Jan]
 
Hsu Zhong Jun
Role: Project Lead [2022 Jan - 2022 Dec]
Committer [2021 June - 2021 Dec]
 
Chan Ger Hean
Role: Committer [2022 Jan - 2022 Dec]
 
Huang Chengyu
Role: Project Co-Lead [2023 Jan - 2023 Apr]
Senior Dev [2023 Jan - 2023 Apr]
Committer [2022 Jan - 2022 Dec]
Contributor [2021 July - 2021 Dec]
 
Chan Jun Da
Role: Project Co-Lead [2023 Apr- 2023 Dec]
Senior Dev [2023 Jan - 2023 Apr]
Committer [2022 Jan - 2022 Dec]
Contributor [2021 July - 2021 Dec]
 
Tay Yi Hsuen
Role: Senior Dev [2023 Jan - 2023 Apr]
Committer [2022 Jan - 2022 Dec]
Contributor [2021 July - 2021 Dec]
 
Zhou Jiahao
Role: Senior Dev [2023 Jan - 2023 Apr]
Committer [2022 Jan - 2022 Dec]
Contributor [2021 July - 2021 Dec]
 ```

2. All the above can be moved to the 'Past Members' section.
 
3. The 4 mentors (Me, David, Marcus, and Gokul), list our current role as 'Senior Developer' for now.

4. Also, please change the end dates of the 'Contributor' role for us 4 mentors so that they end when the 'Senior Developer' role starts.

5. Please add the following person to the current team section.

```
Chang Si Kai
Role: Contributor [2023 Jan - ]
```

**Comment by asdfghjkxd** [comment]
Hi @ckcherry23, I have made the necessary changes following the changes detailed in your previous comment. Please do let me know if I had missed anything out!

**Review by ckcherry23 — APPROVED** [review]
Thanks a lot, looks good to me!

**Review by gok99 — APPROVED** [review]
LGTM!

**Pull request merged**